ci/test/resources/build.yaml

```diff
@@ -46,6 +46,30 @@ steps:
         to: /io/hail-ubuntu
```

Comment:
Can you update the other images to user build args? The `git-make-bash` can take the docker root image as an arg and that serves as a test of build args in inline images.